tests/release_test_data.txt

```diff
@@ -0,0 +1 @@
+{"releases": [{"version": "2021.09", "result": "In Progress", "release_id": 3}, {"version": "2021.07", "result": "Completed", "release_id": 4}]}
```

Comment:
Perhaps we can indent these .txt files so they mirror json outputs. This can further increase readability.